etc/cf-standard-name-table.xml

```diff
@@ -9954,13 +10149,6 @@
       <description>The quantity with standard name mass_fraction_of_rainfall_falling_onto_surface_snow is the mass of rainfall falling onto snow as a fraction of the mass of rainfall falling within the area of interest. The phrase &quot;surface_snow&quot; means snow lying on the surface. Unless indicated in the cell_methods attribute, a quantity is assumed to apply to the whole area of each horizontal grid box.</description>
    </entry>
   
-   <entry id="mass_fraction_of_rain_in_air">
```

Comment:
I assume this change to being an `alias` rather than an `entry` won't negatively affect [the entry in um_cf_map.py](https://github.com/SciTools/iris/blob/f1eb79d41eb232a3eeb6776ab25183a82a3f4db6/lib/iris/fileformats/um_cf_map.py#L187)?

Reply:
Have discussed with @bjlittle , am happy that this is fine